packages/back-end/src/queryRunners/QueryRunner.ts

```diff
@@ -439,14 +438,17 @@ export abstract class QueryRunner<
         (q) => q.status === "running" || q.status === "queued"
       )
     ) {
-      const runningIds = this.model.queries
-        .filter((q) => q.status === "running")
+      const queryIds = this.model.queries
+        .filter((q) => q.status === "running" || q.status === "queued")
         .map((q) => q.query);
```

Comment:
I don't think it makes sense to combine running and queued here since we're doing 2 entirely separate actions.  Create 2 separate arrays instead.

For queued queries, we just need to cancel the timer.  For running, we need to get the queries from Mongo and try to cancel them in the data warehouse.